Functions.js

```diff
@@ -0,0 +1,20 @@
+/* В терминале оплаты сохранено ваше имя, напишите функцию для определения 
+имени в терминале(если вы ввели ваше имя, то привет + имя, если нет, то нет такого имени)*/
+function getAccess() {
+  let currenUserName = prompt("Ваше имя, сэр","");
+  return (currenUserName == "Артем")? ("Привет" + " " + currenUserName) : 'Нет такого имени';
+  };
+console.log(getAccess());
+
+//Напишите функцию вычисления типа аргумента и вывод типа в консоль
+let func = (a) => typeof (a);
+console.log(func(3)) //Number
```

Comment:
1) Logging in console should be inside function `func()`.
2) Get more descriptive name for this function (e.g. `getTypeOfArgument()`).